components/KiboMobileFacetContainer.vue

```diff
@@ -12,7 +12,9 @@
         />
       </div>
       <KiboFilterTiles :applied-filters="appliedFilters" @removeSelectedFilter="removeFilter" />
-      <hr class="filter-hr" />
+      <hr
+        :class="`${className === 'time-filter' ? 'filter-hr filter-hr--time-filter' : 'filter-hr'}`"
```

Comment:
instead passing string classname, we can handle it by boolean value as isTimeFilter to added addition class as "", and filter-hr could be applied directly in both conditions.
```
<hr
        :class="`${'filter-hr--time-filter' : isTimeFilter }`"
        class="filter-hr"
      />
```

Reply:
Resolved